deploy/vagrant/make_boxes/dtaas/README.md

```diff
@@ -1,21 +1,61 @@
-# DTaaS Boxes
+# DTaaS Vagrant Box
 
-This is a box that has the following items:
+This README provides instructions on creating a custom Operating System
+virtual disk for running the DTaaS software. The virtual disk is managed
+by **vagrant**. The purpose is two fold:
 
-* docker
-* nodejs and yarn
-* jupyter
-* microk8s
+* Provide cross-platform installation of the DTaaS application. Any operating system supporting use of vagrant software utility can support installation of the DTaaS software.
```

Comment:
@astitva1905 If the line is broken into two lines, then another error comes:

`Lists should be surrounded by blank lines`

do you know of a good way to resolve this problem?

Reply:
Yes Prasad
Simply shift some part to the next line and then add enough spaces at the beginning of the line so that the indentation matches the indentation of “Provide”. It should work then.